cpp/ps0003/ps0003_soin_cpp.cpp

```diff
@@ -2,11 +2,15 @@
 #include<conio.h>
 using namespace std;
 int main(){
+	// prompting user for number 2/2
 	int number;
 	cin >> number;
+

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number